custom_components/tahoma/sensor.py

```diff
@@ -289,6 +298,20 @@
         value=lambda value: str(value).capitalize(),
         entity_registry_enabled_default=False,
     ),
+    # Various covers
+    OverkizSensorDescription(
+        key="core:Memorized1PositionState",
+        name="Memorized Position",
+        entity_registry_enabled_default=False,
+        icon="mdi:content-save-cog",
+    ),
+    OverkizSensorDescription(
+        key="core:DiscreteRSSILevelState",
+        name="Discrete RSSI Level",
+        entity_registry_enabled_default=False,
+        value=lambda value: str(value).capitalize(),
```

Comment:
Why capitalize ?

Reply:
We do this for other sensors as well. We get ‘low’ as input and I thought that other integrations always capitalize the first letter in a text sensors, thus ‘Low’.

But we can also revert this change throughout the whole integration.

Reply:
That’s fine. I thought it was LOW.